tools/tools.yml

```diff
@@ -373,6 +373,12 @@
       badges: pypi
       builtons: [matplotlib]
 
+    - repo: bjlittle/geovista
+      site: https://geovista.readthedocs.io
+      conda_channel: conda-forge
+      badges: pypi, conda, codecov, site
+      builtons: [pyvista]
```

Comment:
Hey @tkoyama010, if you want pyvista to appear nicely in the `Built on` column of the table you could add its entry to the `builtons.yml` file with its logo in the folder `doc/_static/badges/builtons`.

Reply:
Great idea! I will make it.

Reply:
resolved in 8a98d5d and 7f610a9 .